docs/en/guides/pipeline-integrations/development/sgtk-developer-engine.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Developing engines
+pagename: sgtk-developer-engine
+lang: en
+---
+
+# Developing your own engine
+
+## Introduction
+This document outlines some of the technical details relating to Toolkit engine development.
+
+Table of Contents:
+- [What is a Toolkit engine?](#what-is-a-toolkit-engine)
+- [Things to know before you start](#things-to-know-before-you-start)
+- [Approaches to engine integration](#approaches-to-engine-integration)
+    - [Host application includes Qt, PyQt/PySide and Python](#host-application-includes-qt-pyqtpyside-and-python)
+    - [Host application includes Qt and Python but not PySide/PyQt](#host-application-includes-qt-and-python-but-not-pysidepyqt)
+    - [Host application includes Python](#host-application-includes-python)
+    - [Host application does not contain Python but you can write plugins](#host-application-does-not-contain-python-but-you-can-write-plugins)
+    - [Host application provides no scriptability at all](#host-application-provides-no-scriptability-at-all)
+- [Qt window parenting](#qt-window-parenting)
+- [Startup behavior](#startup-behavior)
+- [Host application wish list](#host-application-wish-list)
+
+## What is a Toolkit engine?
+When developing an engine, you effectively establish a bridge between the host application and the various apps and frameworks that are loaded into the engine. 
+The engine makes it possible to abstract the differences between applications so that apps can be written in more of a software agnostic manner using Python and Qt.
```

Comment:
Great suggestions, I've taken them all except for the connect creation software. I felt "software" was probably enough to distinguish between them, once I'd also change to Toolkit apps.

> When developing an engine, you effectively establish a bridge between the host software and the various Toolkit apps and frameworks that are loaded into the engine. 
The engine makes it possible to abstract the differences between software so that apps can be written in more of a software-agnostic manner using Python and Qt.